cfgov/jinja2/v1/_includes/molecules/expandable.html

```diff
@@ -52,10 +63,12 @@
         </div>
     </button>
     <div class="m-expandable_content"
-         {{ 'aria-expanded=true' if settings.is_expanded else '' }}>
+            {{ 'aria-expanded=true' if value.is_expanded else '' }}>
         <div class="m-expandable_content-animated">
-            {{ caller() }}
+            {{ value.paragraph |safe }}
```

Comment:
Can everything be passed in through `value.optional_content`, including this paragraph? It seems weird to me to have the ability to pass in arbitrary conten, yet separate a paragraph out from that.